airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -1401,24 +1402,18 @@ abstract class BasicFunctionalityIntegrationTest(
                             "id" to 1,
                             "combined_type" to maybePromote("string", "string1"),
                             "union_of_objects_with_properties_identical" to
-                                maybePromote("object", mapOf("id" to 10, "name" to "Joe")),
```

Comment:
These aren't supposed to be promoted, because they are merged into single objects upstream. Were they promoted in the old tests?